### PR TITLE
Admin homepage updated UI

### PR DESCRIPTION
**Pull request by parkjohan**
Updated the Admin Homepage UI to look more modern.
Added simple navigation from Admin Homepage to some blank template screens

### CODE REVIEW

File: plentyofpets/lib/components/admin_homepage_buttons [hunk 1]
```diff
@@ -0,0 +1,134 @@
+import 'package:flutter/material.dart';
```
Comment (jmarr445) [inline plentyofpets/lib/components/admin_homepage_buttons]: Haha I was looking at this wondering why it didn't have syntax highlighting! Could you add the .dart extension type to the file name? 😃

File: plentyofpets/lib/components/admin_homepage_buttons [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+import 'package:admin_homepage_navigation/add_pet.dart';
+
+class AddPetCardButton extends StatelessWidget {
+  final String buttonText;
+
+  AddPetCardButton({required this.buttonText});
```
Comment (jmarr445) [inline plentyofpets/lib/components/admin_homepage_buttons]: There is some convention where things that extend StatelessWidget are supposed to include a parameter of type Key and called super() with the Key. You can take a look [here](https://github.com/fraszle/CS467_PetAdoption/blob/main/plentyofpets/lib/components/decorations.dart). VSCode can actually add some of this boilerplate code for you if you look for the warning highlights

File: plentyofpets/lib/components/admin_homepage_buttons [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+        ),
+      ),
+    );
+  }
+}
+
+class EditPetListCardButton extends StatelessWidget {
+  final String buttonText;
+
+  EditPetListCardButton({required this.buttonText});
```
Comment (jmarr445) [inline plentyofpets/lib/components/admin_homepage_buttons]: Same deal here with the Key parameter and it looks like the other classes as well

File: plentyofpets/lib/components/admin_homepage_buttons [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+
+class AddPetCardButton extends StatelessWidget {
+  final String buttonText;
+
+  AddPetCardButton({required this.buttonText});
+
+  @override
+  Widget build(BuildContext context) {
+    return SizedBox(
+      width: 150,
```
Comment (jmarr445) [inline plentyofpets/lib/components/admin_homepage_buttons]: I like how you've pulled these buttons out into classes. Is this sized box like the border of the button or some background? I'm just worried that these fixed height/width values might not always work well depending on the text that is passed in. Maybe it's possible to use Padding and achieve a similar affect? Something like

`Padding(
    padding: EdgeInsets.all(15),
    child: Card(...)
)`

File: plentyofpets/lib/components/admin_homepage_buttons [hunk 5]
```diff
@@ -0,0 +1,134 @@
...
+                },
+                child: Text(buttonText)),
+          ],
+        ),
+      ),
+    );
+  }
+}
+
+class AddNewsCardButton extends StatelessWidget {
```
Comment (jmarr445) [inline plentyofpets/lib/components/admin_homepage_buttons]: Maybe you don't need to make the change now, but if all these buttons will be styled the same, then these could probably just be one class that also accepts a function parameter for the onPressed property of the TextButton

**Review by jmarr445 — CHANGES_REQUESTED** [review]

File: plentyofpets/lib/components/admin_homepage_buttons (hunk 4 above)
Comment (parkjohan) [reply plentyofpets/lib/components/admin_homepage_buttons]: ah good idea, I'll try to put it inside a Padding to avoid uneven shaped boxes

**Comment by parkjohan** [comment]
Merging this to get some basic UI setup. Will refactor the code to reduce duplications later!

**Pull request merged**